index.html

```diff
@@ -34,7 +34,7 @@
           <a href="#" class="logo">My Logo</a>
         </p>
         <nav class="menu">
-          <img src="./picture/Union.png" alt="" />
+          <img src="./picture/Union.png" alt="Menu-burger" />
```

Comment:
- [ ] Good job 👏 providing an alternative text here but please make sure the `alt` text you use is functional as "Menu-burger" might confuse the average user into thinking it's something eatable 🍔. I think a better one would be "Menubar" 😉.